lang/hr/index.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Semantičko verzioniranje 2.0.0
+language: hr
+---
+
+Semantičko verzioniranje 2.0.0
+==============================
+
+Sažetak
+-------
+
+Uzevši oznake verzije VEĆA.MANJA.ZAKRPA, povećavajte:
```

Comment:
I would put "inkrementirajte", since I would translate "povećavajte" I would translate "increase". I think that the former has a meaning to increase by +1, and the latter increase by some amount.